src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data_clear_experience_configs.py

```diff
@@ -0,0 +1,186 @@
+        of the updated PrivacyExperienceConfig records, post-migration.
+        """
+        bind.execute(
+            text(
+                """
+                DELETE FROM privacyexperienceconfig;
+                """
+            )
+        )
+
+    drop_fk_constraint(bind)
+    delete_experiences(bind)
+    delete_experience_configs(bind)
+
+
+def migrate_notices(bind):
+    """
+    Migrates existing PrivacyNotice records to the new, translation-based data model.
+
+    Notice keys are not impacted. Most existing notice data is moved onto a linked NoticeTranslation record,
+    which is assumed to be english-language as a default. If the existing notice text/content is _not_ english-language,
+    then users will need to *manually* adjust the language on the created translation record.
+
+    The existing PrivacyNoticeHistory records are adjusted to point to the associated NoticeTranslation record,
+    as is expected in the new data model.
+    """
+
+    get_notices_query = text(
+        """SELECT id from privacynotice;
+         """
+    )
+
+    create_notice_translation_query = text(
+        """
+        INSERT INTO noticetranslation (id, privacy_notice_id, language, title, description, created_at, updated_at)
+        SELECT :record_id, :original_notice_id, :language, name, description, created_at, updated_at
+        FROM privacynotice
+        WHERE id = :original_notice_id
+        """
+    )
+    for res in bind.execute(get_notices_query):
+        # Create a Notice Translation for each notice, setting to english as a default
+        bind.execute(
+            create_notice_translation_query,
+            {
+                "record_id": generate_record_id("pri"),
+                "language": "en",
+                "original_notice_id": res["id"],
+            },
+        )
+
+    # Add a FK from the Notice History back to the Notice Translation
+    link_notice_history_to_translation_id = text(
+        """
+        UPDATE privacynoticehistory
+        SET translation_id = noticetranslation.id, title = privacynoticehistory.name
+        FROM noticetranslation
+        WHERE noticetranslation.privacy_notice_id = privacynoticehistory.privacy_notice_id;
+    """
+    )
+
+    bind.execute(link_notice_history_to_translation_id)
+
+
+def downward_migrate_notices(bind):
+    """
+    Back-migration of PrivacyNotice data.
+
+    Moves data in existing NoticeTranslation records back to the associated PrivacyNotice records,
+    to match the old data model.
+    # NOTE: this won't work "well" if multiple translations are present. Should we even attempt it?
+
+    The existing PrivacyNoticeHistory records are adjusted to point back to the associated PrivacyNotice records,
+    rather than the NoticeTranslation records, as is expected in the old data model.
+
+    # TODO: should we attempt to back-migrate region data from the ExperienceConfig records to the PrivacyNotice records? Feels dicey...
```

Comment:
If we're not going to forward migrate them, I don't know if we need to back fill these on the downrev.  I think we need to be taking db snapshots instead for this one.

Reply:
agreed, was leaning that way too but wanted to get another opinion 👍 i'll remove the TODO here.